common/notifications.cpp

```diff
@@ -852,4 +852,129 @@ namespace rs2
 
         _started = true;
     }
+
+    void export_manager::process_flow(
+        std::function<void()> cleanup,
+        invoker invoke)
+    {
+        _progress = 5;
+        _exporter->process(_data);
+        _progress = 100;
+
+        _done = true;
+    }
+
+    void export_notification_model::draw_content(ux_window& win, int x, int y, float t, std::string& error_message)
+    {
+        using namespace std;
+        using namespace chrono;
+
+        ImGui::SetCursorScreenPos({ float(x + 9), float(y + 4) });
+
+        ImVec4 shadow{ 1.f, 1.f, 1.f, 0.1f };
+        ImGui::GetWindowDrawList()->AddRectFilled({ float(x), float(y) },
+        { float(x + width), float(y + 25) }, ImColor(shadow));
+
+        if (update_state != STATE_COMPLETE)
+        {
+            ImGui::Text("Export in progress");
+
+            ImGui::SetCursorScreenPos({ float(x + 10), float(y + 35) });
+
+            ImGui::PushStyleColor(ImGuiCol_Text, alpha(light_grey, 1. - t));
+
+            std::string s = to_string() << "Saving 3D view "
+                << (get_manager().get_data().is<rs2::points>() ? "without texture to " : "to ") <<
```

Comment:
Not clear what "without texture to..  means - also couldn't reproduce this flow

Reply:
You are right, this is redundant today (remains from early versions where you could have pointcloud without texture)